alpa/torch/__init__.py

```diff
@@ -0,0 +1,122 @@
+try:
+    import torch
+except ImportError as e:
+    print(
+        """
+Attempted to use Alpa-PyTorch frontend, but PyTorch is not installed.
+
+Please follow instructions at docs/torch_build_instructions.txt to install PyTorch and related dependencies.
+"""
+    )
+    raise e
+
+from typing import Any, Callable
+
+from torchdistx import deferred_init as torchdistx_deferred_init
+import numpy as np
+import alpa
+from alpa.device_mesh import DistributedArray
+from alpa.torch.ops.utils import is_torch_tensor_type
+
+from . import nn, ops, optim
+from .nn import functionalize
+from .ops.mapping import bind_ops
+from .utils import array_init_like, make_shaped_array_from_pt_tensor
```

Comment:
Use consistent importing style.

In alpa, we use the full paths. e.g.,
https://github.com/alpa-projects/alpa/blob/e9ab27257e188d7a832e29f757e3698e40e25d57/alpa/pipeline_parallel/pipeshard_executable.py#L25